lang/cs/Org.Apache.REEF.IO.Tests/TestHadoopFileSystem.cs

```diff
@@ -142,6 +143,27 @@ public void TestCreate()
             Assert.Throws<NotImplementedException>(() => _fileSystem.Create(GetTempUri()));
         }
 
+        [Fact(Skip = "These tests need to be run in an environment with HDFS installed.")]
+        public void CreateUriForPathNoPrefix()
+        {
+            Uri uri = _fileSystem.CreateUriForPath("/tmp/TestHadoop");
+            Assert.True(new Regex("hdfs://[a-z]+:\\d+/tmp/TestHadoop").Match(uri.AbsoluteUri).Success);
```

Comment:
Why use a regex? Isn't the baseUri explicit?

Reply:
No, it's not. Look at GetUriPrefix() in HadoopFileSystem.cs.

Reply:
I stand corrected.